go/ros/ros2db3_to_mcap.go

```diff
@@ -263,6 +264,12 @@ func DB3ToMCAP(w io.Writer, db *sql.DB, opts *mcap.WriterOptions, searchdirs []s
 	if err != nil {
 		return err
 	}
+
+	progressBar := progressbar.NewOptions64(
+		int64(len(topics)),
```

Comment:
what is len(topics) doing here?

Reply:
isn't number of topics the denominator of the progress?